tests/gnrc_tcp/Makefile

```
@@ -0,0 +1,56 @@
+include ../Makefile.tests_common
+
+# Basic Configuration
+BOARD ?= native
+TAP ?= tap0
+
+# Shorten default TCP timeouts to speedup testing
+MSL_US ?= 1000000
+TIMEOUT_US ?= 3000000
+
+# Mark Boards with insufficient memory
+BOARDS_INSUFFICIENT_MEMORY := arduino-duemilanove arduino-leonardo \
```

```suggestion
BOARD_INSUFFICIENT_MEMORY := arduino-duemilanove arduino-leonardo \
```

Sorry, if that was copy-paste from me ;)